enterprise-suite/scripts/lbc.py

```diff
@@ -464,6 +464,21 @@ def install(creds_file):
 
     helm_args = ''
     if len(args.helm) > 0:
+        # namespace can be passed to helm directly after -- and can take format -namespace <value> or --namespace=<value>
+        # remove namespace from helm args and assign it args.namespace
+        # handle --namespace <val>
+        if  "--namespace" in args.helm and args.helm.index("--namespace") < len(args.helm):
+            ns_index=args.helm.index("--namespace")
+            args.namespace=args.helm[ns_index+1]
```

Comment:
I think we should check to see if there's already an `args.namespace` value that's different.  If so, I'd suggest we bail with a message.

Maybe we shouldn't default the `lbc.py` `namespace` value in the `sub parser.add_argument` call.  That way, `args.namespace` would only have a value at this point if it was explicitly set by the user.  We could then set it to our default (`lightbend`) if need be after we've looked for a `namespace` in the helm args.

Reply:
the new logic, will check if there is namespace value is helm-args, if there is we will suppress the default one.. 
note: the last namespace value wins